### PR TITLE
[improve][doc] SEO for Transactions

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR do SEO for Transactions

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

image
image
image
image
image
image

### CODE REVIEW

File: docs/txn-advanced-features.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: txn-advanced-features
 title: Advanced features
 sidebar_label: "Advanced features"
+description: Get a comprehensive understanding of advanced features with transactions in Pulsar.
```
Comment (Anonymitaet) [inline docs/txn-advanced-features.md:5]:
```suggestion
description: Get a comprehensive understanding of advanced features of transactions in Pulsar.
```

File: docs/txn-how.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 id: txn-how
 title: How transactions work?
 sidebar_label: "Working principles"
+description: Get a comprehensive understanding of how transactions work in Pulsar.
```
Comment (Anonymitaet) [inline docs/txn-how.md:5]:
```suggestion
description: Learn the working principles of transactions in Pulsar.
```

File: docs/txn-why.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 id: txn-why
 title: Why transactions?
 sidebar_label: "Why transactions?"
+description: Get a comprehensive understanding of why develop transaction feature in Pulsar.
```
Comment (Anonymitaet) [inline docs/txn-why.md:5]:
```suggestion
description: Learn the use cases and limitations of transactions in Pulsar.
```

**Comment by zuobiao-zhou** [comment]
updated, PTAL @Anonymitaet

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**